### ISSUE TITLE
Start a New Chapter translations needed

### ISSUE THREAD

**Issue by jcn** [awesomefoundation/awesomebits#405]
We just added a new page explaining the steps for starting a new Awesome Foundation chapter (#403). The page can be found at https://www.awesomefoundation.org/en/start_a_chapter and the text can be found here: https://github.com/awesomefoundation/awesomebits/blob/master/config/locales/en.yml#L237-L262 

We need translations for:

- [ ] bg
- [ ] es
- [ ] fr
- [ ] hy
- [ ] pt
- [ ] ru

**Comment by avicaplan** [comment]
Translation also required for the short paragraph at https://www.awesomefoundation.org/en/about_us that links to this page.

**Comment by jcn** [comment]
> Translation also required for the short paragraph

Thanks, added that above.

**Comment by stroudn1** [comment]
I'd be interested in helping with the 'es' translation. Would the links in the 4 bullets above need translated?

**Comment by jcn** [comment]
@stroudn1 It's actually just 1 page and 1 paragraph (I've updated the bullets). 

Ideally you would work with the translation files themselves - for example the `es` translation is at https://github.com/awesomefoundation/awesomebits/blob/master/config/locales/es.yml and you would copy the lines out of the English file into the es.yml file and then translate them in place. You can either open a pull  request here or just email me the file and I can add it back in for you.

Let me know if you have any questions about the translations or how this process should work.

**Comment by stroudn1** [comment]
Thank you @jcn ! I think I've got it :) One other question - looking at the translations, when you mention a "chapter", I'm assuming that's like a group/division as opposed to a chapter in a book?

**Comment by jcn** [comment]
@stroudn1 That's correct. If you check out the rest of the `es.yml` file, you'll see how we've been translating words, and we should try to be consistent with what's already done. 

If you're unfamiliar with the Awesome Foundation, you can read the about page https://www.awesomefoundation.org/en/about_us to get more of a sense of the context as well.

**Comment by stroudn1** [comment]
Thanks again @jcn ! I've submitted my PR. I followed along with how things were translated as well. I was just curious if I should use 'capítulo' or 'sección' when referring to chapter.
